homeassistant/components/sensor/starlingbank.py

```diff
@@ -44,7 +44,7 @@
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up the Sterling Bank sensor platform."""
-    from starlingbank import StarlingAccount
+    from starlingbank import StarlingAccount  # pylint: disable=syntax-error
```

Comment:
Actually, this is not a syntax error but catches a bad package! 

https://github.com/Dullage/starlingbank/blob/master/starlingbank/__init__.py#L12

It contains invalid syntax and is not compatible with Home Assistant min version of Python 3.5.

Reply:
I've opened a PR to fix the typing https://github.com/Dullage/starlingbank/pull/6 and hope that @Dullage can get a new release added to Home Assistant as soon as possible. 

For now, keep the pylint disable in place.

Reply:
No problems, I've raised a PR with a bump in version that includes @balloob's changes.

https://github.com/home-assistant/home-assistant/pull/21501